uwblueprint/feeding-canadian-kids
# wip 30 login page functionality

## Zenhub ticket link

[Login Page Functionality](https://app.zenhub.com/workspaces/fck-627480c10d8974001872b0ef/issues/gh/uwblueprint/feeding-canadian-kids/30)



## Implementation description
* Added error messages and styling
* Connected frontend to backend auth functions
* Added redirect to sign up page



## Steps to test
1. Navigate to the login page
2. Click the "sign up now" underlined text and ensure that it redirects you to the sign up page.
3. Return to the login page
4. Try logging in using some random combination for username/email. Verify that an error message and error styling pops up.
5. Try logging in using the username `alyssaguo4@uwblueprint.org` and password `test123`. Verify that you get redirected to the home page, and that when you click "Go to dashboard", you are directed to the logged in dashboard rather than the login page.



## What should reviewers focus on?
* Nothing too crazy in this PR, mostly just making sure that the testing works


## Checklist
